extensions/github-pull-requests/package.json

```
@@ -0,0 +1,69 @@
+{
+	"name": "github-repository-search",
+	"title": "GitHub Pull Requests",
+	"description": "Shows Pull Requests where you’re an assignee, a reviewer, or author.",
+	"icon": "icon.png",
+	"author": "rolandleth",
+	"license": "MIT",
+	"commands": [
+		{
+			"name": "index",
+			"title": "Pull Requests",
+			"subtitle": "GitHub",
+			"description": "Shows Pull Requests where you’re an assignee, a reviewer, or author",
+			"mode": "view"
+		}
+	],
+	"preferences": [
+		{
+			"name": "username",
+			"type": "textfield",
+			"required": true,
+			"title": "Username",
+			"description": "Your GitHub username",
+			"placeholder": "Enter your GitHub username"
+		},
+		{
+			"name": "token",
+			"type": "password",
+			"required": true,
+			"title": "Personal Access Token",
+			"description": "Create a token with repository and organization access"
+		},
+		{
+			"name": "ignoredRepos",
+			"type": "textfield",
+			"required": false,
+			"title": "Ignored repositories",
+			"description": "Comma separated repositories, in the form of\norg/repo or user/repo or repo, e.g raycast/extensions or extensions.\nUseful for ignoring certain repos"
+		},
+		{
+			"name": "ignoredAuthors",
+			"type": "textfield",
+			"required": false,
+			"title": "Ignored PR authors",
```

Minor thing to keep the titles consistent
```suggestion
			"title": "Ignored authors",
```

---

The preference above is "ignored repos", so simply writing "ignored authors" might make one think that it's about repo authors, which is why I explicitly wrote "PR".

---

Ah okay, makes sense then.